README.md

```diff
@@ -434,7 +433,7 @@ If you'd like to customize this list, modify the example code in ```config/initi
 
 ```ruby
 config.exception_level_filters.merge!({
-  'ActiveRecord::RecordNotFound' => 'ignore',
+  'ActionController::RoutingError' => 'ignore',
```

Comment:
Oh, sorry, you are right. I think the comment is confusing but right at same time, since if you raise a `ActiveRecord::RecordNotFound` exception in Rails you will have a 404 response code. I'll check this.